roles/openshift_logging/files/elasticsearch-logging.yml

```diff
@@ -28,32 +39,24 @@ logger:
 additivity:
   index.search.slowlog: false
   index.indexing.slowlog: false
+  deprecation: false
 
 appender:
-  console:
```

Comment:
It should be fairly simple to make this a template instead and then we can have some `{% if %} {% endif %}` blocks